AndroidManifest.xml

```diff
@@ -267,11 +263,13 @@
             android:name=".ui.activity.ManageSpaceActivity"
             android:label="@string/manage_space_title"
             android:theme="@style/Theme.ownCloud" />
-        <activity android:name=".ui.activity.PrivacyPolicyActivity"></activity>
-        <activity android:name=".ui.preview.PreviewVideoActivity"
+        <activity android:name=".ui.activity.PrivacyPolicyActivity" />
+        <activity
+            android:name=".ui.preview.PreviewVideoActivity"
             android:configChanges="keyboard|keyboardHidden|orientation|screenSize|screenLayout|smallestScreenSize|uiMode"
-            android:theme="@style/Theme.ownCloud.Fullscreen">
-        </activity>
+            android:theme="@style/Theme.ownCloud.Fullscreen" />
+        <activity android:name=".ui.activity.CapturedFileChecker" />
```

Comment:
Please include the suffix "Activity" for all the activities you add here, it will be helpful to distinguish them from services and other components. In this case, **CapturedFileCheckerActivity** would be great.

Reply:
I have removed the activity tags as there is no such activity present in the project(I created the activity before but then changed my approach and deleted them as they were of no use but the activity tag for them was still present in the Manifest file)